src/entt/entity/view.hpp

```diff
@@ -638,6 +638,23 @@ class basic_view<get_t<Get...>, exclude_t<Exclude...>, std::enable_if_t<(sizeof.
         return iterable{base_type::begin(), base_type::end()};
     }
 
+    /**
+     * @brief Attaches a storage to the end of the view for iterating.
+     *
+     * @tparam OGet The type of the storage to attach to the view.
+     * @param other The storage to attach to the view.
+     * @return A new view with the given storage attached.
+     * @note This function only supports adding one storage, use @ref operator| to join two views instead.
+     * @sa operator|
+     */
+    template<typename OGet, typename = std::enable_if_t<std::is_lvalue_reference_v<OGet>>>
+    [[nodiscard]] auto join(OGet &&other) const noexcept {
```

Comment:
Ehy, sorry for the late reply. I've been busy recently.
I think you can have `OGet &other` and remove the `is_lvalue_reference_v` check as a consequence, no? `remove_reference_t` below is also useless then.
Same for the other `join`. Am I missing something? 👍

Reply:
Huh I'd swear I had a problem with the `const` not propagating from storage to view earlier, I probably did something wrong... Anyway I just reverted the changes, let me know if you need anything else.